src/server/controllers/user.js

```diff
@@ -8,27 +8,42 @@ const jwtSecret = 'mysecret';
 const register = async (req, res) => {
     const { username, password } = req.body;
 
-    const createdUser = null;
-
-    res.json({ data: createdUser });
-};
+        const hashedPassword = await bcrypt.hash(password, 10)
+        const createdUser = await prisma.user.create({
+          data: {
+            username: username,
+            password: hashedPassword
+          },
+        });
+      
+        res.json({ data: createdUser });
+      };
+
+//     const createdUser = null;
+
+//     res.json({ data: createdUser });
```

Comment:
Best to clean the code up if you don't plan to use it before submitting the work.

Reply:
I agree with Brandon's comment about removing the comments when you implement the code for it.